environment.yml

```diff
@@ -36,10 +37,10 @@ dependencies:
   - recommonmark==0.4.0
   - secretstorage
   - sphinx-automodapi
+  - git+https://github.com/iminuit/probfit.git
```

Comment:
This can be removed

Reply:
it is at least used in validate_trigger_digicam.ipynb

Reply:
also in:
digicampipe/calib/camera/charge.py
build/lib/digicampipe/scripts/fmpe.py
build/lib/digicampipe/scripts/spe.py

Reply:
ok then let's keep it. In the long term I want to avoid probfit. In my experience it is slowing down computation